packages/web-components/src/components/content-group-horizontal/__tests__/content-group-horizontal.test.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, render } from 'lit-html';
+import '../../content-item-horizontal/content-item-horizontal';
+import '../content-group-horizontal';
+import { ICON_PLACEMENT } from '../../link-with-icon/link-with-icon';
+import { CTA_TYPE } from '../../cta/shared-enums';
+
+const template = (props?) => {
+  const { children } = props ?? {};
+  return html`
+    <dds-content-group-horizontal>
+      ${children}
+    </dds-content-group-horizontal>
+  `;
+};
+
+describe('dds-content-group-horizontal', function() {
+  describe('Misc attributes', function() {
+    it('should render with minimum attributes', async function() {
+      render(template(), document.body);
+      await Promise.resolve();
+      expect(document.body.querySelector('dds-content-group-horizontal')).toMatchSnapshot({ mode: 'shadow' });
+    });
+
+    it('should render with various attributes', async function() {
+      render(
+        template({
+          children: html`
+            <dds-content-item-horizontal .copy="copy-foo">
+              <slot slot="eyebrow">eyebrow-foo</slot>
```

Comment:
`<slot>` is for defining a slot in shadow DOM, instead of using it. Please refer to: #4406

```suggestion
              <span slot="eyebrow">eyebrow-foo</span>
```